docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -1,77 +1,108 @@
 ---
 id: account-recovery-password-reset
-title: Account recovery and password reset
+title: Recover access to accounts and perform password reset
+sidebar_label: Account recovery
+toc_max_heading_level: 4
 ---
 
-```mdx-code-block
 import RenderFlow from "@site/src/theme/Code/RenderFlow"
-
 import Tabs from "@theme/Tabs"
 import TabItem from "@theme/TabItem"
-```
 
 Account recovery allows registered users to regain access to their account if they forget their password or lose access to the
 second authentication factor required for multifactor authentication (MFA).
 
 To regain access to their account, the user must prove that they are the owner of the account. The verification of ownership is
-performed using the recovery address defined by the user. This could be an email address or a phone number. When the account
-recovery flow is initiated, the system sends a link or a one-time code to the recovery address defined by the user. The user must
-access the link or enter the code they received to confirm their identity and ownership of the account.
+performed using the [recovery address defined by the user](#account-recovery-address). When the account recovery flow is
+initiated, the system sends a link or a one-time code to the recovery address defined by the user. The user must access the link
+or enter the code they received to confirm their identity and ownership of the account.
+
+When the user accesses the configured recovery method, they receive a
+[privileged session](../../session-management/overview#privileged-sessions) and are taken to their account's settings page where
+they must create a new password to regain access to their account.
+
+:::info
+
+The account recovery flow doesn't reset user's second authentication factor.
```

Comment:
This is a bit too vague, IMO, as it can absolutely be used to reset the second authentication factor. After all, the user is redirected to the settings page, that can be used to change the second factor.

What you probably mean is that, by default, the recovery flow does not require the second factor to be presented after entering the recovery code. The [part I wrote about that](https://github.com/ory/docs/pull/995/files/d4cd72fe889dd648a8a0ae307eb54edb018066ae..e453d5ed18c36812de26064ac6ed732af3152df5#diff-8d79b4961459ffe8dcfeb33b986c91b1bb486fb349c9dc6690f1edb49ef1f69eL205) is just gone without replacement here. What should we do about that?

Reply:
Yes, please re-add that!

Reply:
Why I removed that - enabling 2FA requirement for settings isn't something that's directly tied to the recovery flow. If you require 2FA, the user must provide their 2nd auth factor even if they're not doing the recovery. That's what I got from your comments/the UI text in the Console.